auto_submit/lib/validations/revert.dart

```diff
@@ -0,0 +1,147 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:auto_submit/model/auto_submit_query_result.dart' as auto;
+import 'package:auto_submit/validations/validation.dart';
+import 'package:github/github.dart' as github;
+import 'package:auto_submit/service/github_service.dart';
+import 'package:github/github.dart';
+
+import '../service/config.dart';
+import '../service/log.dart';
+
+class Revert extends Validation {
+  Revert({
+    required Config config,
+  }) : super(config: config);
+
+  static const Set<String> allowedReviewers = <String>{ORG_MEMBER, ORG_OWNER};
+
+  /// Validate a revert pull request.
```

Comment:
Nit: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation

Reply:
Cleaned up all docs in this file.